### PR TITLE
Beginning of quantum section to docs

### PR DESCRIPTION
**Pull request by LeahHuk**

### CODE REVIEW

File: quantum/frequently_asked_quantum_questions.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+
+How do I access the OLCF quantum computing resources? 
+====================================================
+Applications for both Quantum Computing projects and quantum user accounts can be found 
+on the quantum accounts documentation :ref:`quantum-accounts` page.
+
+
+What happens after I apply for access to QCUP? 
+=============================================
```
Comment (michael-sandoval) [inline quantum/frequently_asked_quantum_questions.rst:37]: Need one more "=", it needs to match the heading length

File: quantum/frequently_asked_quantum_questions.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+Applications are put through a merit review process, and you will be contacted 
+regarding the status of your application. See the user documentation page for more details.
+
+
+I formerly had access to quantum resources, but my backends/lattices/etc. have disappeared, what do I do? 
+=======================================================================================================
```
Comment (michael-sandoval) [inline quantum/frequently_asked_quantum_questions.rst:44]: Need two more "=", it needs to match the heading length

File: quantum/frequently_asked_quantum_questions.rst [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+If your account was established prior to July 5th, 2020, and was not through the OLCF directly, 
+your access to quantum resources has been removed, and you will need to re-apply to an OLCF project. 
+Also, if your access to your OLCF project or the project access itself has expired, you will also 
+need to reapply.
+
+
+I applied to a quantum computing resource via the vendor website, but don’t have access; what do I do? 
+====================================================================================================
```
Comment (michael-sandoval) [inline quantum/frequently_asked_quantum_questions.rst:53]: Need two more "=", it needs to match the heading length

File: quantum/quantum_access.rst [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+User groups utilize quantum computing expertise to investigate diverse application interests, 
+using multiple programming languages, quantum-classical programming, and multiple software 
+environments. Most projects focus on proof-of-principle demonstrations and/or new method 
+development. Some projects focus on application performance and/or benchmarking, and 
+additionally some projects focus on device characterization, verification, and validation.
+
+
+
+Request a New Project Allocation
+******************************
```
Comment (michael-sandoval) [inline quantum/quantum_access.rst:53]: Need two more "*", it needs to match the heading length

**Review by michael-sandoval — CHANGES_REQUESTED** [review]
Needs a few formatting changes so it doesn't scream warnings when building. Other than that looks good. Some of the heading formatting symbols need to be changed but I can handle that in another commit

File: quantum/frequently_asked_quantum_questions.rst (hunk 3 above)
Comment (LeahHuk) [reply quantum/frequently_asked_quantum_questions.rst]: fixed

File: quantum/quantum_access.rst (hunk 4 above)
Comment (LeahHuk) [reply quantum/quantum_access.rst]: fixed

File: quantum/frequently_asked_quantum_questions.rst [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+(itself a portmanteau of “binary digit”), consisting of a two-state quantum mechanical 
+system that can (like a classical bit) exist in one state, `|0⟩`, or the other, `|1⟩`, 
+but unlike the classical bit counterpart, a qubit can also be in a quantum superposition 
+of both states.
+
+
+How do I access the OLCF quantum computing resources? 
+====================================================
+Applications for both Quantum Computing projects and quantum user accounts can be found 
+on the quantum accounts documentation :ref:`quantum-accounts` page.
```
[the remark this answers is not in the document]
Comment (LeahHuk) [reply quantum/frequently_asked_quantum_questions.rst]: fixed

[the remark this answers is not in the document]
Comment (LeahHuk) [reply quantum/frequently_asked_quantum_questions.rst]: fixed again

**Review by michael-sandoval — APPROVED** [review]
Looks good now, will change other things later

**Review by chrisfuson — APPROVED** [review]

**Pull request merged**